src/facade/reply_builder_test.cc

```diff
@@ -205,8 +205,8 @@ RedisReplyBuilderTest::ParsingResults RedisReplyBuilderTest::Parse() {
 
 TEST_F(RedisReplyBuilderTest, MessageSend) {
   // Test each message that is "sent" to the sink
-  builder_->SendOk();
-  ASSERT_EQ(TakePayload(), kOKMessage);
+  // builder_->SendOk();
```

Comment:
todo: to bring it back or restore?

Reply:
Yes, actually I'll be simpler to just revert this commit

Reply:
but you have not,  @dranikpg ?